docker/zms/conf/zms.properties

```diff
@@ -9,7 +9,7 @@
 
```

Comment:
Please add the following.
https://github.com/yahoo/athenz/blob/377b2963d3ecf118d2e77663175078bc6d98a4b5/servers/zms/conf/zms.properties#L419-L421

Reply:
thx, fixed in 39efe30